Examples/PublisherExampleSwiftUI/PublisherExampleSwiftUI/View/Map/Map.swift

```diff
@@ -0,0 +1,42 @@
+import SwiftUI
+import MapKit
+
+struct Map: View {
+    
+    @Binding private var center: CLLocationCoordinate2D
+    @State private var span: MKCoordinateSpan?
+
+    private var region: Binding<MKCoordinateRegion> {
+        Binding {
+            let span = span ?? MKCoordinateSpan(
+                latitudeDelta: 0.2,
+                longitudeDelta: 0.2
+            )
+            
+            return MKCoordinateRegion(center: center, span: span)
+        } set: { region in
+            DispatchQueue.main.async {
+                span = region.span
+                center = region.center
+            }
+        }
+    }
+    
+    init(center: Binding<CLLocationCoordinate2D>) {
+        _center = center
+    }
+    
+    var body: some View {
+        MapKit.Map(
+            coordinateRegion: region,
+            showsUserLocation: true
+        )
+    }
+}
+
+struct Map_Previews: PreviewProvider {
```

Comment:
This struct isn't used anywhere, can be removed?

Reply:
@JakubJankowski This is [used by Xcode to display view previews](https://developer.apple.com/documentation/swiftui/previewprovider).

Reply:
Ohh I see, disregard then!